app/controllers/survey-controller.js

```diff
@@ -21,7 +21,7 @@ router.get('/:type/:ref', (req, res, next) => {
         .then(survey => {
           const options = new OptionsWithLabels(survey.get('definition')).options;
 
-          res.render('survey', {
+          res.render(req.params.type, {
```

Comment:
Controller need to be able to handle `ryws` type. Either duplicate controller or add parametrization. This need to be here.